terraform/modules/hub/analytics.tf

```diff
@@ -90,3 +90,65 @@ resource "aws_ecs_service" "analytics" {
     ]
   }
 }
+
+resource "aws_ecs_task_definition" "analytics_fargate" {
+  family                   = "${var.deployment}-analytics-fargate"
+  container_definitions    = data.template_file.analytics_task_def.rendered
+  execution_role_arn       = module.analytics_ecs_roles.execution_role_arn
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = 256
+  memory                   = 512
+}
+
+resource "aws_ecs_service" "analytics_fargate" {
+  name            = "${var.deployment}-analytics"
+  cluster         = aws_ecs_cluster.fargate-ecs-cluster.id
+  task_definition = aws_ecs_task_definition.analytics_fargate.arn
+
+  desired_count                      = var.number_of_apps
+  deployment_minimum_healthy_percent = 50
+  deployment_maximum_percent         = 100
+
+  load_balancer {
+    target_group_arn = aws_lb_target_group.ingress_analytics.arn
```

Comment:
I don't remember if this is allowed... ISTR some horrible restriction that only one service could "own" a target group

Reply:
after reading the docs, it looks like stuff has changed since I last looked at this. It's now possible for a single service to register with multiple target groups, so who knows it might also be possible for a single target group to host multiple services.

Reply:
I had this fear too... but I couldn't spot anything in docs, so was gonna try and see